python/cloud-run-python-hello-world/.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/python-hello-world.iml" filepath="$PROJECT_DIR$/.idea/python-hello-world.iml" />
```

Comment:
This needs to be adjusted in all relevant places to be _cloud-run-python-hello-world_ instead of python-hello-world.

Reply:
Done in next commit.